cob_sound/action/Play.action

```diff
@@ -0,0 +1,6 @@
+# goal
+string filename
```

Comment:
add a comment:
- this is the fully qualified filename including file ending, example: "/home/user/sounds/hello.wav"